modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,134 @@
+= About Multi-Factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts. Non-SSO users can also manually enable multi-factor authentication in their profile settings.
+
+SSO users configure multi-factor authentication through their identity provider.
+
+After multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of four verification methods:
+
+* Third-party TOTP authenticator apps
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+
+== Multi-Factor Authentication Verification Methods
+
+You can configure at least one of the following verification methods to comply with multi-factor authentication requirements:
+
+Third-party TOTP authenticator apps:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID or Windows Hello, to verify your identity when logging in to Anypoint Platform.
+
+Security key:: Registers a USB security key, such as Yubico YubiKey or Google Titan Security Key, to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable or Exempt Multi-Factor Authentication for Your Organization Users
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users without a verification method already configured are prompted to do so when they subsequently log in to Anypoint Platform. Users with a method already configured are not prompted to configure another verification method.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform (known as _service accounts_). You must either configure these service account calls by using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
```

Comment:
```suggestion
To prevent service disruptions, identify any accounts (_service accounts_) that are used to programmatically call Anypoint Platform. You must either configure these service account calls by using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
```